configure/operator.md

```diff
@@ -78,9 +78,11 @@ The ``backup`` section in the [deploy/cr.yaml](https://github.com/Percona-Lab/pe
 |enabled                         | boolean    | `false`   | Enables or disables the backups functionality |
 |version                         | string     | `0.2.1`   |                                               |
 |restartOnFailure                | boolean    | `true`    |                                               |
-|s3.secret                       | string     | `my-cluster-name-backup-s3`| [Kubernetes imagePullSecret](https://kubernetes.io/docs/concepts/configuration/secret/#using-imagepullsecrets) for backups |
-|s3.bucket                       | string     |           | The [Amazon S3 bucket](https://docs.aws.amazon.com/en_us/AmazonS3/latest/dev/UsingBucket.html) name for backups                    |
-|s3.region                       | string     |`us-west-2`| The [AWS region](https://docs.aws.amazon.com/en_us/general/latest/gr/rande.html) to use |
+|storages.type                   | string     | `s3`      | Type of the cloud storage to be used for backups. Currently only `s3` type is supported                                                          |
+|storages.s3.credentialsSecret   | string     | `my-cluster-name-backup-s3`| [Kubernetes imagePullSecret](https://kubernetes.io/docs/concepts/configuration/secret/#using-imagepullsecrets) for backups |
```

Comment:
it is not imagePullSecret :( we can mention that AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY should exist in secret